requirements.txt

```diff
@@ -1,4 +1,4 @@
-dbt-core~=0.18.0
+git+https://github.com/fishtown-analytics/dbt.git
```

Comment:
@mikaelene maybe it's because I'm installing this way which _should_ take the latest commit from `dev/kiyoshi-kuromiya` because it is the default branch. Maybe this is because you were installing pre-release cut on PyPI? feel free to change this to whatever you like.